zebra/zebra_nhg.c

```diff
@@ -381,12 +388,17 @@ static void *zebra_nhg_hash_alloc(void *arg)
 uint32_t zebra_nhg_hash_key(const void *arg)
 {
 	const struct nhg_hash_entry *nhe = arg;
+	uint32_t val, key = 0x5a351234;
+
+	val = nexthop_group_hash(&(nhe->nhg));
+	if (nhe->backup_info) {
```

Comment:
nit: no {}

Reply:
I think that's fixed in the current version